core/app/models/spree/calculator/default_tax.rb

```diff
@@ -16,7 +16,6 @@ def compute_order(order)
       line_items_total = matched_line_items.sum(&:discounted_amount)
       if rate.included_in_price
         order_tax_amount = round_to_two_places(line_items_total - ( line_items_total / (1 + rate.amount) ) )
-        refund_if_necessary(order_tax_amount, order.tax_address)
```

Comment:
Is this going to do the right thing with this removed? If so, the assignment on the previous line should be removed

Reply:
Yeah, it will. And also, yes the assignment is unnecessary. I'm not sure we need to ship this entire method any longer, either, as it we don't do taxation on orders since Spree 2.2.

Reply:
Amended!